src/commands/contract/interfaces.ts

```diff
@@ -125,7 +123,7 @@ export function generateActionsNamespace(abi: ABI.Def): ts.ModuleDeclaration {
 
     const actionParamsTypes = ts.factory.createModuleDeclaration(
         [ts.factory.createModifier(ts.SyntaxKind.ExportKeyword)],
-        ts.factory.createIdentifier('Types'),
+        ts.factory.createIdentifier('Base'),
```

Comment:
Naming this namespace Base doesn't really make sense to me 🤔 How about using _Types instead?

Reply:
Not sure I like `_Types`, since in my testing I actually used some of those things and `_` typically means "Private". 

I'd agree that maybe a better name could be found though.

Reply:
Wait why do we even need to sub-namespace them?

Can't they all just be `ActionParams`?

Reply:
We can, but there would be a risk of naming collision happening since you could have an action name that is named the same as a type.

Reply:
I guess we could use `Type` singular 😅

Reply:
> We can, but there would be a risk of naming collision happening since you could have an action name that is named the same as a type.

Ah alright - true. 

It could be `ActionParams.Types`, and then whenever we use one of them we're just super verbose and use `ActionParams.Types.Thing` instead of just `Thing`. I feel like that would let us use `Types` twice?

Reply:
That doesn't seem to work unfortunately because when you try to use one of the types that is in the root `Types` namespace, typescript seems to assume that you are trying to reference one of the types under `ActionParams.Types` :(